<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>constraintQWindow</class>
 <widget class="QWidget" name="constraintQWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="restPositionQBox">
     <property name="title">
      <string>Rest Position:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="restTranslateQLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Translate:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="restPositionQCheck">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="restRotateQLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Rotate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QDoubleSpinBox" name="restPositionTrXQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="restPositionTrYQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="restPositionTrZQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QDoubleSpinBox" name="restPositionRtXQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="restPositionRtYQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="restPositionRtZQSpin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::PlusMinus</enum>
        </property>
        <property name="minimum">
         <double>-9999.000000000000000</double>
        </property>
        <property name="maximum">
         <double>9999.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="targetOffsetQScroll">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="targetOffsetQWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>263</width>
        <height>69</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="interpTypeQBox">
     <property name="title">
      <string>Interp Type:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QComboBox" name="interpTypeQCombo">
        <item>
         <property name="text">
          <string>No flip</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Average</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Shortest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Longest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cache</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>restPositionQCheck</tabstop>
  <tabstop>restPositionTrXQSpin</tabstop>
  <tabstop>restPositionTrYQSpin</tabstop>
  <tabstop>restPositionTrZQSpin</tabstop>
  <tabstop>restPositionRtXQSpin</tabstop>
  <tabstop>restPositionRtYQSpin</tabstop>
  <tabstop>restPositionRtZQSpin</tabstop>
  <tabstop>targetOffsetQScroll</tabstop>
  <tabstop>interpTypeQCombo</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
